lib/plugins/features/movies/controller.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const Movie = require('../../../models/movie');
+
+// defining code that is shared accross different files. Important to include so that the funciton or the variable can be used outside of this file
+exports.create = (payload) => {
+  // saves the apyload as the movie object in the database
+  return new Movie().save(payload).then( (movie) => {
```

Comment:
Also in this case, you can use the arrow notation that is just 
```
() => <something to return>
```
More specifically,
```
.then((movie) => new Movie({ id: movie.id }).fetch())
```